### PR TITLE
Am feature live text updating

### PR DESCRIPTION
**Pull request by anama118118**

### CODE REVIEW

File: BYT/Utility/FoaasBuilder.swift [hunk 1]
```diff
@@ -57,6 +65,13 @@ class FoaasPathBuilder {
    - returns: A `String` that corresponds to the path component needed to create a `URL` to request a `Foaas` object
    */
```
Comment (spacedrabbit) [inline BYT/Utility/FoaasBuilder.swift:66]: please update the documentation to match the changes to this function

**Review by spacedrabbit — CHANGES_REQUESTED** [review]

File: BYT/Utility/FoaasBuilder.swift (hunk 1 above)
Comment (anama118118) [reply BYT/Utility/FoaasBuilder.swift:66]: Oka, it's updated.

**Review by singhhari — APPROVED** [review]
👍🏼

**Comment by liamdkane** [comment]
👍

**Comment by seymotom** [comment]
👍

**Review by spacedrabbit — APPROVED** [review]

**Pull request merged**